chore: update user voyage roles in dev seed

Ajen07 opened the pull request:
# Description

- Created `2 new users` and voyage `52` and `53`
- Added team `v52-tier1-team-1` to have only `PO` with   `memberId `as `41` and `42`
- Added team `v52-tier1-team-2` to have only `SM`  with `member Id`  `43` and `44`
- Added team `v52-tier2-team-2` to have  `3 developers` with `memberId` `45`, `46` and `47`
- Added team `v52-tier2-team-4`  to have    `1 UI/UX designer`  , `1 data scientist` and `1 voyage guide`  having `memberId `as `48`, `49` and `50`
- Added team `v52-tier3-team-3`  to have  all roles `1 SM`, `1 PO`,` 3 developers`, `1 UI/UX designer` , `1 data scientist`  and `1 voyage guide` having `memberId `as  `51`  , `52`, `53`, `54`, `55`, `56`, `57` and `58`



## Issue link

[Fixes # (issue)](https://app.clickup.com/t/86b27xvk2)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Feature updates / changes
- [ ] Tests
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update


# How Has This Been Tested?

- tested with `yarn seed`
- tested with `yarn test:docker`



# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have updated the change log

cherylli reviewed (commented):
I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.

Ajen07 commented:
> I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.

In that case we will have people  belonging to different teams in the same voyage

cherylli commented:
> > I think it's easier if all the teams with different composition is in the same voyage, so we don't have to manually overwrite the dates to check, but I need to check if frontend can handle that first.
> 
> In that case we will have same people belonging to different teams in the same voyage or we have to create more users.

yes I prefer just using the same members, so we don't have to create too many, but lets see what frontend says. Eventually they will have to handle this situation, but I don't know if the current frontend can handle it yet

JoshuaHinman reviewed (dismissed):
all tests passed

cherylli commented:
@Ajen07, I'd say just go ahead and make them all in the same voyage, v52, 53 sounds good since they are coming up. I just saw the current seed has people in more than 1 team anyway

Ajen07 commented on prisma/seed/voyage-teams.ts:
```diff
@@ -1050,6 +1050,487 @@ export const populateVoyageTeams = async () => {
 
+    //v52
+
+    // team with only PO
```
Ok I will do the necessary changes

JoshuaHinman reviewed (dismissed):
All tests passed!

cherylli commented on prisma/seed/voyage.ts:
```diff
@@ -88,4 +88,34 @@ export const populateVoyages = async () => {
+                },
+            },
+            startDate: new Date("2025-01-06T17:00:00.000Z"),
+            endDate: new Date("2025-02-16T04:59:59.000Z"),
```
please update endDate, soloprojectDeadline, CertificationIssueDate and showcasePublishDat as above (v52) - you can see here for more detail (https://github.com/chingu-x/chingu-dashboard-be/wiki/Timezone)

cherylli reviewed (changes requested):
Please also update the wiki page with the new teams
https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users

cherylli commented:
update second reviewer to @curtwl as josh is away

cherylli reviewed (changes requested):
> Please also update the wiki page with the new teams https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users

Still need this updated

I fixed the voyage dates based on the voyage schedule, it is very important we have the date and exact time right as it would impact what pages/features is enabled/disabled in the frontend

Ajen07 commented:
> > Please also update the wiki page with the new teams https://github.com/chingu-x/chingu-dashboard-be/wiki/Test-Teams-and-Users
> 
> Still need this updated
> 
> I fixed the voyage dates based on the voyage schedule, it is very important we have the date and exact time right as it would impact what pages/features is enabled/disabled in the frontend

I have updated the wiki page

cherylli commented:
Updated v53 start date as it was an invalid date.
Seeding worked fine

curtwl reviewed (approved):
lgtm
